sdk/src/main/scala/com/horizen/SidechainApp.scala

```diff
@@ -67,6 +67,7 @@ class SidechainApp @Inject()
   )
   extends Application  with ScorexLogging
 {
+  // cannot replace ScorexLogging with LazyLogging since Application already inherits from ScorexLogging
```

Comment:
Good idea to keep logger initialization in Settings as soon as we've read the docs.
This comment can be removed.

Reply:
Done